CreateProduct/function.json

```diff
@@ -14,5 +14,5 @@
       "name": "$return"
```

Comment:
it looks like you've deleted the files (instead of moving them)

Reply:
check now!